src/aig-common/old-common/services/cpv.service.ts

```diff
@@ -0,0 +1,37 @@
+import { Injectable } from '@angular/core';
```

Comment:
delete this file. for contact the back end you need use client library. 
CpvResourceService class